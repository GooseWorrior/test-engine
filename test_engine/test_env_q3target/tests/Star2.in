a*

a
aa
b
ab
q
ab*

a
ab
abb
ac
b
q
a*b

b
ab
aab
cb
a
q
a**

a
aa
b
q
ab**

a
ab
abb
ac
b
q
a**b

b
ab
aab
cb
a
q
*a

a
ab
q
*a*
a
aa
b
q
**a**
a
aa
b
q
a*b*

a
b
ab
aab
abb
aabb
c
ca
bc
q
a*b*c*

abc
a
b
c
aabbcc
abc
ab
bc
ac
ba
ca
q
