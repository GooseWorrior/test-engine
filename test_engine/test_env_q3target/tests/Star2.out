Star (a)
true
true
true
false
false
Concatenation (a,Star (b))
false
true
true
true
false
false
Concatenation (Star (a),b)
false
true
true
true
false
false
Star (Star (a))
true
true
true
false
Concatenation (a,Star (Star (b)))
false
true
true
true
false
false
Concatenation (Star (Star (a)),b)
false
true
true
true
false
false
Concatenation (Star (),a)
false
true
false
Concatenation (Star (),Star (a))
true
true
false
Concatenation (Star (Star ()),Star (Star (a)))
true
true
false
Concatenation (Star (a),Star (b))
true
true
true
true
true
true
true
false
false
false
Concatenation (Star (a),Concatenation (Star (b),Star (c)))
true
true
true
true
true
true
true
true
true
true
false
false
